src/components/button-toggle/button-toggle.component.tsx

```diff
@@ -202,7 +202,7 @@ export const ButtonToggle = ({
         onBlur={handleBlur}
         onClick={handleClick}
         onKeyDown={handleKeyDown}
-        {...(tabbable ? {} : { tabIndex: -1 })}
+        {...(tabbable ? { tabIndex: 0 } : { tabIndex: -1 })}
```

Comment:
comment: might be worth a comment about why explicitly setting the tabindex is necessary here